components/home/CurrentCycle.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import {
+  TouchableOpacity, View,
+} from 'react-native';
+import styled from 'styled-components/native';
+import PropTypes from 'prop-types';
+
+const NameText = styled.Text`
```

Comment:
Is that symbol after Text, the apostrophe looking thing meant to be there?

Reply:
yeah, it's what encapsulates the content underneath, there's another ` at the end as well